pdbfixer/pdbfixer.py

```diff
@@ -1195,18 +1202,23 @@ def _createForceField(self, newTopology, water):
                 forcefield._templateSignatures[signature] = [template]
         return forcefield
 
-    def _findNearestDistance(self, context, topology, newAtoms):
+    def _findNearestDistance(self, context, topology, newAtoms, cutoff, exclusions):
         """Given a set of newly added atoms, find the closest distance between one of those atoms and another atom."""
 
         positions = context.getState(getPositions=True).getPositions(asNumpy=True).value_in_unit(unit.nanometer)
-        atomResidue = [atom.residue for atom in topology.atoms()]
-        nearest = sys.float_info.max
+        boxSize = np.max(positions, axis=0)-np.min(positions, axis=0)
+        boxVectors = [(boxSize[0], 0, 0), (0, boxSize[1], 0), (0, 0, boxSize[2])]
+        cells = app.modeller._CellList(positions, cutoff, boxVectors, False)
+        nearest2 = sys.float_info.max
```

Comment:
It wasn't obvious to me that this means "nearest squared", particularly since in `addMissingAtoms` we do have `thing2` variables where the 2 just means 2. I would consider renaming `nearest2` and `dist2` to `nearest_squared` and `dist_squared`.